README.md

```diff
@@ -237,7 +237,7 @@ Here is a partial list of examples:
 - Integrating with services.
   - Mozilla offers some useful services such as
     [linting](https://github.com/mozilla/addons-linter) and
-    [signing](https://addons-server.readthedocs.io/en/latest/topics/api/signing.html)
+    [signing](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign)
```

Comment:
The link should point to the AMO API docs:

```suggestion
    [signing](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html)
```

Reply:
Thanks!